src/css/custom.css

```diff
@@ -21,8 +21,19 @@
   h2 {
     font-size: 1.5rem;
   }
+
+  .navbar {
+    background: rgba(255, 179, 212, 0.8);
+    backdrop-filter: blur(25px);
+    background-image: url('assets/trianglesNavbar.png');
+    background-position: bottom;
+    background-size: auto 100%;
+    background-repeat: repeat-x;
+    filter: hue-rotate(0deg) saturate(.5);
```

Comment:
This will make all elements' saturation decreased. Should we think of another solution of just remove this line?

Reply:
Line 32 was being directly ripped off from the osu! site, originally used for changing homepage's colour scheme (using `hue-rotate`). The original `trianglesNavbar.png` asset contain red colour tone, since I have photoshoped to remove that red colour, line 32 can be deleted.

I have commented this line. No significant difference can be observed after doing this.